src/main/java/it/gov/pagopa/payhub/activities/activity/classifications/IufClassificationActivityImpl.java

```diff
@@ -0,0 +1,89 @@
+package it.gov.pagopa.payhub.activities.activity.classifications;
+
+import it.gov.pagopa.payhub.activities.dao.ClassifyDao;
+import it.gov.pagopa.payhub.activities.dao.PaymentsReportingDao;
+import it.gov.pagopa.payhub.activities.dao.TreasuryDao;
+import it.gov.pagopa.payhub.activities.dto.classifications.ClassifyDTO;
+import it.gov.pagopa.payhub.activities.dto.classifications.IufClassificationActivityResult;
+import it.gov.pagopa.payhub.activities.dto.paymentsreporting.PaymentsReportingDTO;
+import it.gov.pagopa.payhub.activities.dto.treasury.TreasuryDTO;
+import it.gov.pagopa.payhub.activities.exception.PaymentsClassificationSaveException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+/**
+ * Interface for defining an activity to process payment reporting classifications based on IUF.
+ */
```

Comment:
javadoc is not required on implementation, it's inherited from the interface

Reply:
done